FluentTerminal.Models/ShellProfile.cs

```diff
@@ -42,8 +42,8 @@ protected ShellProfile(ShellProfile other)
         public int TabThemeId { get; set; }
         public LineEndingStyle LineEndingTranslation { get; set; }
         public Dictionary<string, string> EnvironmentVariables { get; } = new Dictionary<string, string>();
-
-        public bool UseConPty { get; set; }
+        // Setting ConPTY as default
+        public bool UseConPty { get; set; } = true;

```

Comment:
Can we move this to to DefaultValueProvider? That's where the other defaults are stored

Reply:
Or is this for adhoc profiles? In that case I would set this where it is actually used

Reply:
This line is removed completely - it turned out it isn't needed (it's already set in the constructor).